### PR TITLE
Sorting option in core,email and router routes.

### PR DESCRIPTION
**Pull request by sdimitris**
**What kind of change does this PR introduce?** 

- [ ] Bugfix
- [x] Feature
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other (please describe)

**Does this PR introduce a breaking change?** 

- [ ] Yes
- [x] No

**The PR fulfills these requirements:**

- [x] It's submitted to the `main` branch
- [ ] When resolving a specific issue, it's referenced in the PR's description  (e.g. `fix #xxx`, where "xxx" is the issue number)

Routes which support sorting option:
- /config/modules
- /router/routes
- /router/middlewares
- /email/externalTemplates

### CODE REVIEW

**Review by kkopanidis — CHANGES_REQUESTED** [review]
if only sort by name is available, name it such. The param should be sortByName and the value should be true or false or something simple in general.

File: modules/router/src/admin/router.ts [hunk 1]
```diff
@@ -1,15 +1,21 @@
-import ConduitGrpcSdk, { UnparsedRouterResponse } from '@conduitplatform/grpc-sdk';
+import ConduitGrpcSdk, {
+  GrpcError,
```
Comment (ChrisPdgn) [inline modules/router/src/admin/router.ts:2]: Remove unused import

File: modules/router/src/admin/router.ts [hunk 2]
```diff
@@ -1,15 +1,21 @@
...
+  ParsedRouterRequest,
+  UnparsedRouterResponse,
+} from '@conduitplatform/grpc-sdk';
 import { isNil } from 'lodash';
 import ConduitDefaultRouter from '../Router';
+import { status } from '@grpc/grpc-js';
```
Comment (ChrisPdgn) [inline modules/router/src/admin/router.ts:8]: and here

File: packages/core/src/config-manager/admin/routes/GetModules.route.ts [hunk 3]
```diff
@@ -4,15 +4,22 @@ import {
   ConduitError,
   ConduitString,
   ConduitBoolean,
+  GrpcError,
```
Comment (ChrisPdgn) [inline packages/core/src/config-manager/admin/routes/GetModules.route.ts:7]: and here

**Review by ChrisPdgn — APPROVED** [review]

**Review by kkopanidis — APPROVED** [review]

**Pull request merged**